rkt/rkt.go

```diff
@@ -221,7 +228,9 @@ func main() {
 	// cmdRkt.GenBashCompletionFile("dist/bash_completion/rkt.bash")
 	// os.Exit(0)
 
-	cmdRkt.Execute()
+	if err := cmdRkt.Execute(); err != nil && cmdExitCode == 0 {
```

Comment:
One improvement that this points out is that it would be nice if cobra itself had an option to abort the program itself on errors. Also it is weird that cobra returns a nil error if there is no runnable command (e.g. running rkt without any arguments), internally it treats `rkt` and `rkt help` as identical commands despite the first being a usage error and the second being valid.